03/Task03.js

```diff
@@ -1,25 +1,24 @@
-import React from 'react';
-
-import { Row, Col, Breadcrumb as RBBreadcrumb } from 'react-bootstrap';
+import React from "react";
 
+import { Row, Col, Breadcrumb as RBBreadcrumb } from "react-bootstrap";
+import Breadcrumb from "./../src/components/BreadCrumb/Breadcrumb";
 const Task03 = () => {
-    return (
-        <Row>
-            <Col>
-                <RBBreadcrumb>
-                    <RBBreadcrumb.Item href="#">Home</RBBreadcrumb.Item>
-                    <RBBreadcrumb.Item href="https://getbootstrap.com/docs/4.0/components/breadcrumb/">
-                        Library
-                    </RBBreadcrumb.Item>
-                    <RBBreadcrumb.Item active>Data</RBBreadcrumb.Item>
-                </RBBreadcrumb>
-            </Col>
-            <Col>
-                Breadcrumb!
-            </Col>
-        </Row>
-)
-}
+  return (
+    <Row>
+      <Col>
+        <RBBreadcrumb>
+          <RBBreadcrumb.Item href="#">Home</RBBreadcrumb.Item>
+          <RBBreadcrumb.Item href="https://getbootstrap.com/docs/4.0/components/breadcrumb/">
+            Library
+          </RBBreadcrumb.Item>
+          <RBBreadcrumb.Item active>Data</RBBreadcrumb.Item>
+        </RBBreadcrumb>
+      </Col>
+      <Col>
+        <Breadcrumb></Breadcrumb>
```

Comment:
Tutaj powinieneś przekazywać dane. 
W tym przypadku każda zmiana w ścieżce będzie Cię zmuszać do zmiany kodu.
To rozwiązanie jest nie do wykorzystania na realnej stronie.

Reply:
wlasnie o to pytalem na Slacku, pisales, ze mam w tej kwestii dowolnosc, ale ok, juz poprawione :)

Reply:
W takim razie nie zrozumieliśmy - wybacz